inference-engine/src/gna_plugin/gna_graph_compiler.cpp

```diff
@@ -1804,6 +1810,11 @@ void GNAGraphCompiler::PWLPrimitive(InferenceEngine::CNNLayerPtr layer) {
         activation_type = GNAFakeQuantizeLayer(layer).parseAsActivation();
     }
 
+    if (it->second == kActKaldiLstmClipping) {
+        auto& clamp_layer = dynamic_cast<ClampLayer&>(*layer.get());
```

Comment:
Wouldn't it be better to cast to pointer and check if casting was successful?

Reply:
fixed as recommended